File: foodie-app/src/model/order.ts
export type NewOrder = {
  orderId: number;
  dateOfOrder: any;
  timeOfOrder: any;
  noOfItems: number;
  amount: number;
  menu: any;
  address: {
    street: string | any;
    city: string | any;
    houseNo: string | any;
    pinCode: string | any;
  };
};

File: FoodApplication/OrderService/src/main/java/com/niit/OrderService/domain/Order.java
package com.niit.OrderService.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.awt.*;
import java.util.Date;
import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class Order {
    @Id
    private String orderId;
    private String dateOfOrder;
    private String timeOfOrder;
    private int noOfItems;
    private String amount;
    private List<Menu> menu;
    private Address address;
}
